float64 rpm1
float64 rpm2
float64 rpm3
float64 rpm4